package.json

```diff
@@ -45,6 +52,8 @@
     "ember-cli-inject-live-reload": "^1.8.2",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-template-lint": "^1.0.0-beta.1",
+    "ember-cli-typescript": "^3.0.0",
```

Comment:
Please remove these! Since this add-on is not being built with TS, including them is entirely unnecessary. Simply by having the type definition file in the right place, with the correct body in that file, all TS consumers (and any JS users using the TS language server, in fact!) will get the definitions working correctly out of the box.